src/SpaceTime/2d/Acceleration2.cs

```diff
@@ -0,0 +1,272 @@
+
+        #endregion
+
+        #region operators
+
+        #region algebra
+
+        /// <summary>
+        /// Adds two acceleration vectors.
+        /// </summary>
+        public static Acceleration2 operator +(Acceleration2 a0, Acceleration2 a1)
+        {
+            return new Acceleration2(a0.value + a1.value);
+        }
+        /// <summary>
+        /// Subtracts a acceleration vector from another.
+        /// </summary>
+        public static Acceleration2 operator -(Acceleration2 a0, Acceleration2 a1)
+        {
+            return new Acceleration2(a0.value - a1.value);
+        }
+
+        #endregion
+
+        #region scaling
+
+        /// <summary>
+        /// Inverts the acceleration vector.
+        /// </summary>
+        public static Acceleration2 operator -(Acceleration2 a)
+        {
+            return new Acceleration2(-a.value);
+        }
+        /// <summary>
+        /// Multiplies the acceleration vector with a scalar.
+        /// </summary>
+        public static Acceleration2 operator *(Acceleration2 a, float scalar)
+        {
+            return new Acceleration2(a.value * scalar);
+        }
+        /// <summary>
+        /// Multiplies the acceleration vector with a scalar.
+        /// </summary>
+        public static Acceleration2 operator *(float scalar, Acceleration2 a)
+        {
+            return new Acceleration2(a.value * scalar);
+        }
+        /// <summary>
+        /// Divides the acceleration vector by a divisor.
+        /// </summary>
+        public static Acceleration2 operator /(Acceleration2 a, float divisor)
+        {
+            return new Acceleration2(a.value / divisor);
+        }
+
+        #endregion
+
+        #region ratio
+
+        /// <summary>
+        /// Divides a acceleration vector by a speed, returning an untyped vector.
+        /// </summary>
+        public static Vector2 operator /(Acceleration2 a, Acceleration divisor)
+        {
+            return a.value / divisor.NumericValue;
+        }
+
+        #endregion
+
+        #region integrate
+
+        /// <summary>
+        /// Multiplies a acceleration vector by a timespan, returning a velocity vector.
+        /// </summary>
+        public static Velocity2 operator *(Acceleration2 a, TimeSpan t)
+        {
+            return new Velocity2(a.value * (float)t.NumericValue);
+        }
+        /// <summary>
+        /// Multiplies a acceleration vector by a timespan, returning a velocity vector.
+        /// </summary>
+        public static Velocity2 operator *(TimeSpan t, Acceleration2 a)
+        {
+            return new Velocity2(a.value * (float)t.NumericValue);
+        }
+
+        #endregion
+
+        #region comparision
+
+        /// <summary>
+        /// Compares two acceleration vectors for equality.
+        /// </summary>
+        public static bool operator ==(Acceleration2 a0, Acceleration2 a1)
+        {
+            return a0.value == a1.value;
+        }
```

Comment:
I think it would be more consistent with how equals and comparison operators work if this operator would call the `Equals` method which has the implementation. The only reason to do it this way is to prevent an extra method call as overhead, but that could probably be mitigated by aggressive inlining.